model/src/cmake/src_list.cmake

```diff
@@ -3,6 +3,7 @@ set(c_src w3getmem.c)
 # Core files always built
 set(ftn_src
   constants.F90
+  mpicomm.F90
```

Comment:
I did try to include ifdefs to make mpicomm.F90 not affect anything if it was compiled without MPI, but I can try to change this to [model/src/cmake/switches.json](https://github.com/NOAA-EMC/WW3/pull/1277/files#diff-964aadbc838d0deb6acde149cab012d78dc268b318f933f8dcb0a09da51396f1) since that sounds cleaner